docs/elevate/ELevate-quickstart-guide.md

```diff
@@ -2,112 +2,115 @@
 title: "ELevate Quickstart Guide"
 ---
 
-###### last updated: 2024-07-08
+###### last updated: 2024-08-07
 
 # ELevate Quickstart Guide
 
 ::: warning
 Before beginning, we **HIGHLY** recommend that you follow system administration best practices and make sure you have backups and/or snapshots of your system before you proceed. It is recommended to do a trial run in a sandbox to verify the upgrade worked as expected before you attempt to upgrade any production system. Please report any issues encountered to the [AlmaLinux Bug Tracker](https://bugs.almalinux.org) and/or [AlmaLinux Chat Migration Channel](https://chat.almalinux.org/almalinux/channels/migration)
 :::
 
-This guide contains steps on how to upgrade your RHEL-based operating system to the next major version.
+This guide contains steps on how to upgrade your RHEL-based operating system to the next major version using the Leapp tool version **0.19.0**.
```

Comment:
leapp version is 0.16.0
leapp-repository version is 0.19.0

But it's not clear why we should mention version here, let's remove.

Reply:
removed versions